### PR TITLE
Update root readme

### PR DESCRIPTION
**Pull request by JonathanGiles**
This is a draft update of the root azure-sdk-for-java readme.md file based on discussions today. It follows a standard template that was agreed upon, so I am getting this out for review earlier in case there is more feedback that should be worked through in the template before other languages attempt to update their readme.md file. I have slightly extended the template by including links to docs.microsoft.com, otherwise this is essentially unchanged. The only link that currently does not work is for storage blobs, but it should light up once the preview is made available.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,145 +1,50 @@
...
-5. Create new Pull Request
-
----
-
-This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/). For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
+# Azure SDK for Java
 
+[![Build Status](https://dev.azure.com/azure-sdk/public/_apis/build/status/17?branchName=master)](https://dev.azure.com/azure-sdk/public/_build/latest?definitionId=17) [![Build Documentation](https://img.shields.io/badge/documentation-published-blue.svg)](https://azuresdkartifacts.blob.core.windows.net/azure-sdk-for-java/index.html)
 
-![Impressions](https://azure-sdk-impressions.azurewebsites.net/api/impressions/azure-sdk-for-java%2FREADME.png)
```
Comment (weshaggard) [inline README.md]: Can you please add this back to the bottom of the readme? It is use for tracking how stats for how much this is readme is viewed.

File: README.md [hunk 2]
```diff
@@ -1,145 +1,50 @@
...
+The libraries released in the July 2019 preview:
+- [App Configuration](appconfiguration/client/README.md)
+- [Event Hubs](eventhubs/client/README.md)
+- [Key Vault Keys](keyvault/client/keys/README.md)
+- [Key Vault Secrets](keyvault/client/secrets/README.md)
+- [Storage Blobs](storage/client/README.md)
+
+>NOTE: If you need to ensure your code is ready for production, use one of the stable libraries.
+
+
```
Comment (conniey) [inline README.md]: nit: extra blank line.

File: README.md (hunk 1 above)
Comment (JonathanGiles) [reply README.md]: Will do - good reminder also for @kaerm, @bterlson, @KrzysztofCwalina and @johanste to do the same (with their tracking IDs).

File: README.md [hunk 3]
```diff
@@ -1,145 +1,50 @@
...
+## Packages available
+Each service might have a number of libraries available from each of the following categories:
+
+* [Client - July 2019 Preview](#Client-July-2019-Preview)
+* [Client - Stable](#Client-Stable)
+* [Management](#Management)
+
+### Client: July 2019 Preview
+New wave of packages that we are currently releasing in **preview**. These libraries allow you to use and consume existing resources and interact with them, for example: upload a blob. These libraries share a number of core functionalities such as http retries, logging, transport protocols, authentication protocols, etc. that can be found in the [azure-core](core) library. You can learn more about these libraries by reading the guidelines that they follow [here](https://azuresdkspecs.z5.web.core.windows.net/JavaSpec.html). Library directories start with `azure-`, e.g. `azure-keyvault`.
```
Comment (conniey) [inline README.md]: nit: HTTP rather than http.

Comment (JonathanGiles) [reply README.md]: Fixed

File: README.md (hunk 2 above)
Comment (JonathanGiles) [reply README.md]: Fixed

File: README.md [hunk 4]
```diff
@@ -1,145 +1,50 @@
...
-
----
-
-This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/). For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
+# Azure SDK for Java
 
+[![Build Status](https://dev.azure.com/azure-sdk/public/_apis/build/status/17?branchName=master)](https://dev.azure.com/azure-sdk/public/_build/latest?definitionId=17) [![Build Documentation](https://img.shields.io/badge/documentation-published-blue.svg)](https://azuresdkartifacts.blob.core.windows.net/azure-sdk-for-java/index.html)
 
-![Impressions](https://azure-sdk-impressions.azurewebsites.net/api/impressions/azure-sdk-for-java%2FREADME.png)
+This repository contains official Java libraries for Azure services. For reference documentation go to [Azure SDK for Java documentation](http://aka.ms/java-docs), and tutorials, samples, quick starts and other documentation, go to [Azure for Java Developers](https://docs.microsoft.com/java/azure/).
```
Comment (conniey) [inline README.md]: I wish these lines had a line break at the 120 character mark (or some other mark). It makes viewing, adding comments, and editing these a lot easier. (I don't have line-wrap on when I'm editing files, so the horizontal scrolling is not so fun when you're trying to select multiple lines in vim.)
Visual Studio Code extensions like [rewrap](https://marketplace.visualstudio.com/items?itemName=stkb.rewrap) allow you to do it easily.

https://about.gitlab.com/2016/10/11/wrapping-text/

**Pull request merged**